src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -34,14 +35,16 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(@RequestParam Long movieSessionId, Authentication authentication) {
         shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+                movieSessionService.get(movieSessionId),
+                userService.get(userService.findByEmail(authentication.getName()).get().getId()));
```

Comment:
```suggestion
                userService.findByEmail(authentication.getName()).get()));
```

I think you get the User two times